src/modules/test/unit-tests/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+project(testfactorytests)
+
+include(CTest)
+find_package(GTest REQUIRED)
+
+add_executable(testfactorytests MimParserTests.cpp TestRecipeParserTests.cpp)
+target_include_directories(testfactorytests PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/..)
+target_link_libraries(testfactorytests gtest gtest_main pthread testfactorylib)
+gtest_discover_tests(testfactorytests)
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/../../mim/commandrunner.json DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/mim/)

```

Comment:
Is it possible to use some glob to copy all MIM JSON files?

Reply:
there is not going to ever be more MIMs here, im expecting the sample module to have all types needed

Reply:
Please do not use the Command Runner or any other production MIM for this. Use either a test MIM or the sample MIM (and again please note: "sample" not "samples". There is only one sample MIM and code sample that gets replicated in several languages but from MIM perspective there is only 1 (one) sample MIM.

Reply:
Since this is only for the unit tests it may also be useful to distinguish that this sample is a C++ sample (`src/modules/samples/cpp`). There will likely be other samples in the future.